### PR TITLE
Add new and updated licenses

### PR DESCRIPTION
**Pull request by AyanSinhaMahapatra**
Thanks for the new licenses and updates @dennisclark

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁
* [x] Looked for possible updates in documentation and added updates if applicable
* [x] Updated CHANGELOG.rst

### CODE REVIEW

File: src/licensedcode/data/licenses/niels-ferguson.LICENSE
Comment (DennisClark) [inline src/licensedcode/data/licenses/niels-ferguson.LICENSE:1]: @AyanSinhaMahapatra    The original owner `Niels Ferguson` was correct and should be restored, or you can simply drop this change altogether.  Thanks.

**Review by DennisClark — CHANGES_REQUESTED** [review]
@AyanSinhaMahapatra   everything looks good, except please do not change the owner of niels-ferguson "Niels Ferguson License" to Unspecified but leave it as "Niels Ferguson".  Thanks.

File: src/licensedcode/data/licenses/niels-ferguson.LICENSE
Comment (AyanSinhaMahapatra) [reply src/licensedcode/data/licenses/niels-ferguson.LICENSE:1]: @DennisClark done, thanks!

**Review by DennisClark — APPROVED** [review]
@AyanSinhaMahapatra    Looks ready to deploy, thanks.

**Pull request merged**